starknet_py/tests/e2e/conftest.py

```diff
@@ -141,19 +168,55 @@ def address_and_private_key(pytestconfig):
 def gateway_account_client(address_and_private_key, gateway_client):
     address, private_key = address_and_private_key
 
-    return create_account_client(address, private_key, gateway_client)
+    return create_account_client(
+        address, private_key, gateway_client, supported_tx_version=0
+    )
 
 
 @pytest.fixture(scope="module")
 def rpc_account_client(address_and_private_key, rpc_client):
     address, private_key = address_and_private_key
 
-    return create_account_client(address, private_key, rpc_client)
+    return create_account_client(
+        address, private_key, rpc_client, supported_tx_version=0
+    )
 
 
 @pytest.fixture(scope="module")
-def account_clients(gateway_account_client, rpc_account_client):
-    return gateway_account_client, rpc_account_client
+def new_address_and_private_key(pytestconfig):
+    net = pytestconfig.getoption("--net")
+
+    account_details = {
+        # TODO configure for other envs
+        "devnet": None,
+        "testnet": None,
```

Comment:
How does it work?

Reply:
It is not. In the current test setup, we don't use new_account_client, because transaction version = 1 is not guaranteed to work correctly on devnet

Reply:
Working on updating tests to use accounts deployed by us

Reply:
Oh I see, so this would be a seperate PR?

Reply:
Yes, I think it will be clearer than adding this here